Dockerfile

```diff
@@ -176,3 +175,7 @@ ADD requirements.txt /tuna/
 WORKDIR /tuna
 
 RUN python3 setup.py install
+
+# Install SolverAnalytics (github.com/ROCmSoftwarePlatform/SolverAnalytics#installation)
+ARG SOLVER_ANALYTICS_DIR=/tuna/SolverAnalytics/
```

Comment:
i think you might have to create the dir /tuna/SolverAnalytics and then try and clone in there. What is the error you are getting?